### PR TITLE
[plese delete] Recipe for WMS Watchdog

### PR DESCRIPTION
**Pull request by klugem**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This is my first recipe for bioconda. If I can improve something, please let me know.
Watchdog: https://github.com/klugem/watchdog

### CODE REVIEW

**Comment by klugem** [comment]
Seems to me that there is a problem with travis ci. Other pull request fails with the same error.

File: recipes/watchdog-wms/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  build:
+  run:
+      - openjdk >=8
+
+extra:
+  maintainers:
+   - Michael Kluge <michael.kluge@bio.ifi.lmu.de>
+  notes: |
+     Graphical user interface works only with javafx, which is currently not part of openjdk8.
+     Dependency should be changed to java-jdk when working again.
```
Comment (mbargull) [inline recipes/watchdog-wms/meta.yaml]: Even though there is an older `java-jdk` package available, that is OpenJDK as well, IIRC.
So we'd actually need someone to create a new package for [OpenJFX](http://wiki.openjdk.java.net/display/OpenJFX/Main) over at `conda-forge`.

Comment (klugem) [reply recipes/watchdog-wms/meta.yaml]: Yes, I had that included in my local version but automated build failed caused by java-jdk dependency.

**Comment by klugem** [comment]
For some reason there is a problem with my test script. It builds when I run /tmp/miniconda/bin/conda-build on my local installation but it fails when travis is used...I'll try to identify the problem.

**Comment by klugem** [comment]
I commited a file I don't want to have in the repo and recreated my fork...

**Comment by klugem** [comment]
I commited a file I don't want to have in the repo and recreated my fork...

**Pull request closed without merging**